nodejs14.x/author-from-scratch/cdk/cdk/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "lambda-from-scratch-cdk",
+    "version": "0.0.1",
+    "bin": {
+        "cdk": "bin/cdk.js"
+    },
+    "scripts": {
+        "build": "tsc",
+        "watch": "tsc -w",
+        "test": "jest",
+        "cdk": "cdk"
+    },
+    "devDependencies": {
+        "@aws-cdk/assert": "^1.31.0",
+        "@types/jest": "^25.2.1",
+        "@types/node": "^10.17.5",
```

Comment:
I think this needs to be updated to `14.17.5`, right?  (Same for other `package.json` files)